docs/data-management.rst

```diff
@@ -568,7 +568,7 @@ Previously created Shares can still be viewed in the ``Shares`` tab |image6| abo
  - You will also be sharing the data with all the other members of that group
 
 .. note::
-    Shares were never supported by OMERO.iviewer. In case OMERO.iviewer is installed on your server and is set as a default viewer, it is not possible for the users with whom the images inside a historical Share are shared to view these in a Full viewer.
+    Shares were never supported by OMERO.iviewer. If OMERO.iviewer is installed on your server and is set as a default viewer, then, for the Share participants, it is not possible to view images inside a historical Share in any Full viewer.
```

Comment:
I think something like
``Users with permissions to access the data in the Share cannot view images`` might be easier to understand

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/24/commits/da4c6dacc94c9ff024fc20a693780e517b97321e